### ISSUE TITLE
Missing alt text for Farah's image

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#1326]
**URL**
https://alpha.austin.gov/police-oversight
https://alpha.austin.gov/es/police-oversight

**Screenshots**
![image.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/5b533cb7-39b6-4431-b36f-6e3d96e92eb7)

**Comment by amenity** [comment]
Duplicate of #1314.
